casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -121,8 +135,10 @@ class RuntimeManagerImpl[F[_]: Concurrent] private[rholang] (
     val startHash = emptyStateHash
     withRuntimeLock { runtime =>
       for {
+        span       <- Metrics[F].span(computeGenesisLabel)
         _          <- setBlockTime(blockTime, runtime)
-        evalResult <- processDeploys(startHash, terms, processDeploy(runtime))
+        _          <- span.mark("before-process-deploys")
```

Comment:
...for example here you are calling span.mark here because you don't want to pass the span in processDeploys. 

Because of this you have to duplicate the mark message